spec/queries/claims/claims_query_spec.rb

```diff
@@ -8,10 +8,11 @@
   describe "#call" do
     it "returns all visible claims, ordered by created at date descending" do
       _internal_claim = create(:claim)
-      draft_claim = create(:claim, :draft, created_at: Date.parse("29 March 2024"))
+      _draft_claim = create(:claim, :draft, created_at: Date.parse("29 March 2024"))
+      create(:claim, :draft, created_at: Date.parse("29 March 2024"))
```

Comment:
Is there a reason we have to create 2 draft claims here? Seems like we can achieve the same thing with the `_draft_claim`?